yarn-project/stdlib/src/tx/tx_request.ts

```diff
@@ -16,18 +16,19 @@ export class TxRequest {
   constructor(
```

Comment:
Order of params in constructor didn't follow order of params in `TxRequest` noir struct which led to a failing test once the Noir's intrinsic serialization was used. I am just aligning the ordering here with what's in Noir.

Reply:
My body is so ready for this errors to be banished forever. Nice work!!!